src/lib/swapper/swappers/LifiSwapper/utils/getLifiChainMap.ts

```diff
@@ -1,15 +1,15 @@
 import type { ChainId as LifiChainId, ChainKey } from '@lifi/sdk'
 import type { ChainId } from '@shapeshiftoss/caip'
-import { fromChainId } from '@shapeshiftoss/caip'
+import { arbitrumNovaChainId, fromChainId } from '@shapeshiftoss/caip'
 import { evmChainIds } from '@shapeshiftoss/chain-adapters'
 
 import { createLifiChainMap } from './createLifiChainMap/createLifiChainMap'
 import { getLifi } from './getLifi'
 
 export const getLifiChainMap = async (): Promise<Map<ChainId, ChainKey>> => {
-  const supportedChainRefs = evmChainIds.map(
-    chainId => Number(fromChainId(chainId).chainReference) as LifiChainId,
-  )
+  const supportedChainRefs = evmChainIds
```

Comment:
feels like an explicit `LifiChainId[]` const would be ideal, but this works and removes arbitrum nova which is not supported as described here: https://li.quest/v1/chains?chainTypes=EVM.

we could also have a dynamic supported list using that endpoint and cross referencing our supported evm chains in the future.

Reply:
As discussed, happy to revisit (we can even have this dynamic as you've spotted) in a follow-up, this PR was just to make this specific request happy as a first step to figuring out what was wrong with Li.Fi! 

<img width="1418" alt="image" src="https://github.com/shapeshift/web/assets/17035424/dcc74dea-958e-467b-b60b-e24e87000402">